filters/include/pcl/filters/crop_hull.h

```diff
@@ -64,11 +69,15 @@ namespace pcl
       using Ptr = shared_ptr<CropHull<PointT> >;
       using ConstPtr = shared_ptr<const CropHull<PointT> >;
 
-      /** \brief Empty Constructor. */
-      CropHull () :
+      /** \brief Constructor.
+        * \param[in] extract_removed_indices Set to true if you want to be able to extract the indices of points being removed (default = false).
+        */
+      CropHull (bool extract_removed_indices = false) :
```

Comment:
This constructor should be `explicit` to avoid weird implicit conversion issues. Ideally it also shouldn't use a default parameter but have a separate default constructor declared as `CropHull(): CropHull(false) {}` which makes the converting constructor `explicit` without making the default constructor `explicit`.

Reply:
applied suggestion